### ISSUE TITLE
Add key with Ledger

### ISSUE THREAD

**Issue by hanakannzashi** [near/near-cli-rs#163]
It would be convenient if we can add a public key with Ledger.
At present I can only view public key in Ledger via old near-cli, and then add key by manual provided public key.

**Comment by frol** [comment]
@hanakannzashi Fair point! Would you like to give it a try implementing it? We are currently working on [near-social CLI extension](https://github.com/FroVolod/near-social), so this feature request won't be addressed soon.

**Comment by frol** [comment]
Here is the starting point where you would need to add `UseLedgerPublicKey`:

https://github.com/near/near-cli-rs/blob/b0d50210a53ac4c2972c173927bec871665a6f24/src/commands/account/add_key/mod.rs#L62-L81

, and then use the example of `use-manually-provided-public-key`, just create a new folder following the pattern [here](https://github.com/near/near-cli-rs/tree/master/src/commands/account/add_key).

You can find useful code snippets [here](https://github.com/near/near-cli-rs/blob/b0d50210a53ac4c2972c173927bec871665a6f24/src/commands/account/create_account/fund_myself_create_account/add_key/use_ledger/mod.rs) (e.g. how to get the public key from Ledger)

**Comment by frol** [comment]
Just for context, currently, when I try to add a key to my existing account, I have 3 options:

<img width="978" alt="image" src="https://user-images.githubusercontent.com/304265/234265045-5aac9006-ec40-41b8-af02-79e9507ba9c5.png">

Command to reproduce:

```
cargo run -- account add-key frol.near grant-full-access
```

* Generate a key on my host system (not secure, as private key will be in the RAM)
* Provide a seed phrase (also not secure, as you reveal the private key)
* Provide a public key (secure if you store your private key securely, e.g. you might have extracted the public key from your Ledger device)

The proposal suggests automating the public key extraction from a Ledger device, so add the fourth variant `use-ledger`, similarly to the existing [adding key flow when a new account is created](https://github.com/near/near-cli-rs/blob/59b78ab5339b7235f42d1fd7bb90f3d3e1f5bfc2/src/commands/account/create_account/fund_myself_create_account/add_key/mod.rs#L31-L34):

<img width="1451" alt="image" src="https://user-images.githubusercontent.com/304265/234266739-5f31c958-7ee4-4dbc-8d90-15b2eb709dbc.png">

Command to see the menu:

```
cargo run -- account create-account fund-myself new.testnet '0.1 NEAR'
```

**Comment by frol** [comment]
@hanakannzashi Resolved in #188. I will cut a new release later this week.
